tools/ear/make_EAR.xml

```diff
@@ -282,6 +298,26 @@ NOTES:
                     <param name="hap2_exists_curated" value="no"/>
                 </conditional>
             </section>
+            <section name="method_data">
```

Comment:
Why is this twice here? Should it be a macro?

Reply:
There are two different parameters in the `methods_data` and are not repetitive

Reply:
macros can be parameterized

Reply:
Okay, have made the change